### ISSUE TITLE
Hanging signs

### ISSUE THREAD

**Issue by GoSuperKebab** [acranum/ChestShop#2]
Will this plugin be updated to support hanging signs?

**Comment by GoSuperKebab** [comment]
Yes, that’s right, it wouldn’t make sense for player shops but for Admin Shops it’d be useful imo
________________________________

**Comment by acranum** [comment]
Hey, thanks for your feedback! 👍
I've been inactive for a while for personal reasons, but I've recently found some time to implement your suggestion. Now the hanging signs for adminshops should work in the latest version

Even if this comes a little late I hope I could help you a little further
